meteor/client/lib/triggers/TriggersHandler.tsx

```diff
@@ -514,7 +514,7 @@ export const TriggersHandler: React.FC<IProps> = function TriggersHandler(
 								keys: hotkeyTriggers,
 								finalKeys: hotkeyFinalKeys,
 								name: pair.name,
-								targetName: adLib.label,
+								targetName: pair.useLabelAsName && pair.name ? pair.name : adLib.label,
```

Comment:
In this context `useLabelAsName` seems to be used wrong. From the naming, I would assume that when `useLabelAsName` is checked the label should be used, otherwise the name, but it is the other way around here.

Reply:
You are right about it seems wrong in this context. The thing is... `pair.name` actually is the label that we can see in the UI
![image](https://user-images.githubusercontent.com/23049647/204208365-5090e196-f0b0-464d-91ab-ec0f54af3314.png)
![image](https://user-images.githubusercontent.com/23049647/204208934-6ad04644-f064-42c0-bef1-b6c523e1bdf4.png)

So we are going to have a mismatch somewhere.
But since the column for the "label" actually is called `name` in the database I think renaming `useLabelAsName` would be the best approach. For the name to make sense in this context it would have to be something like `useNameForTarget`, but I'm afraid that that would not make sense when looking at the attribute in the database. There something like `useNameForDisplay` would be better, but you would think that the default behaviour was to use name for display.
I'm open for suggestions

Reply:
Ahh, I see. I don't have a good solution, but the naming in the db vs the UI is misleading.